migration-to-PERCS-v2.0.md

```diff
@@ -0,0 +1,222 @@
+symbol, separated by a space.
+So, for a method of a subclass that does nothing:
+
+    <?php
+    class SubClass extends BaseClass
+    {
+        protected function init() {}
+    }
+
+## [Section 4.6](https://www.php-fig.org/per/coding-style/#46-modifier-keywords)
+
+Modifier keywords are keywords that alter how PHP handles classes,
+properties and methods.
+
+These keywords MUST BE ordered as follows:
+
+[abstract|final] [public|protected|private] [static] [readonly]
+[type] name
+
+Furthermore all keywords must be on a single line and MUST be separated
+by a single space.
+
+## [Section 4.7](https://www.php-fig.org/per/coding-style/#47-method-and-function-calls)
+
+Named arguments and method chaining
+
+When making a method or function call, there MUST NOT be a space between
+the method or function name and the opening parenthesis, there MUST NOT
+be a space after the opening parenthesis, and there MUST NOT be a space
+before the closing parenthesis.
+
+In the argument list, there MUST NOT be a space before each comma,
+and there MUST be one space after each comma.
+
+Argument lists MAY be split across multiple lines, where each subsequent
+line is indented once. When doing so, the first item in the list MUST be
+on the next line, and there MUST be only one argument per line. A single
+argument being split across multiple lines (as might be the case with a
+closure or array) does not constitute splitting the argument list itself.
+
+If using named arguments, there MUST NOT be a space between the argument
+name and colon, and there MUST be a single space between the colon and
+the argument value.
+
+
+## [Section 4.8](https://www.php-fig.org/per/coding-style/#48-function-callable-references)
+
+Function callable references - there must not be whitespace surrounding the '...' operator ()
+
+
+## [Section 5.2](https://www.php-fig.org/per/coding-style/#52-switch-case-match)
+The match keyword is now covered 
+
+
+## [Section 7.1](https://www.php-fig.org/per/coding-style/#71-short-closures)
+
+A new subsection about Short Closures.
+
+As with standard closures, 'fn' must not be succeeded by a space.
+The '=>' symbol MUST be preceeded and succeeded by a space. (No leading or trailing spaces)
+The semicolon at the end MUST NOT be preceeded by a leading space.
+The expression portion MAY be split to a subsequent line. If so, the => MUST be 
+included on the second line, and MUST be indented once.
+
+
+## [Section 9](https://www.php-fig.org/per/coding-style/#9-enumerations)
+
+Enums - enums, or rather enumerations are a layer on top of classes and class
+constants that provide a way to define a very limited set of possible values
+for a custom type. Enums are new to PHP since 8.1 and so could not be covered
+by PSR12.
+
+PER-CS addresses aspects on how standardise on coding Enums in new section 9.
+As Enums do not support inheritance non-public methods MUST use the private
+keyword.
```

Comment:
Most of this exposition is not necessary.  Let's keep it short.  "Enums are now covered."  Just like `match` didn't bother to describe inline as it's a totally new section, Enums are a totally new section that we can just link straight to.  If we want to include anything else here, it should be just a bulleted list and/or a short example.

Reply:
Agreed